cmd/collector/app/builder/builder_flags.go

```diff
@@ -38,4 +38,6 @@ var (
 	CollectorPort = flag.Int("collector.port", 14267, "The tchannel port for the collector service")
 	// CollectorHTTPPort is the port that the collector service listens in on for http requests
 	CollectorHTTPPort = flag.Int("collector.http-port", 14268, "The http port for the collector service")
+	// HealthCheckHTTPPort is the port that the health check service listens in on for http requests
+	HealthCheckHTTPPort = flag.Int("health-check.http-port", 15268, "The http port for the health check service")
```

Comment:
In environments like OpenShift and Kubernetes, it's easier to protect ports than paths. For instance, this port could be kept for internal communication only, while the main HTTP port can be exposed as a "route".